src/table/cells/handle-key-press.js

```diff
@@ -115,7 +114,12 @@ export const bodyHandleKeyPress = (evt, rootElement, rowIndex, colIndex, cell, s
     // Enter: Confirms selections.
     case 'Enter': {
       preventDefaultBehavior(evt);
-      selState.api.confirm();
+      selState.api.confirm().then(() => {
+        setTimeout(() => {
+          const { rowElements } = getRowAndColumnCount(rootElement);
+          moveToNextFocus(rowElements, 1, 0);
+        }, [400]);
+      });
```

Comment:
Unfortunately, we can't rely on this. If the engine takes longer than 400ms than we lose focus anyway. Maybe we can have some sort of state that tells the chart to set focus when it rerenders, e.g. `shouldRefocus`. But then setting that fo false again might trigger a new reload and then it looses focus. Another alternative is that nebula handles that. So whenever the selections are confirmed, focus is set to the entire chart

Reply:
True. The engine! 
It goes like that the table receives an 'Entre' key event and calls the API. The engine confirms the selections. Then the nebula calls the component() again so the table rerenders and gets the latest table data and etc.
I don't know if we are able to let the table set focus after the table finishes rerendering.
Can we know the rerender of the table is caused by confirming selections?

Reply:
If you set some state on the TableWrapper only for when confirming with enter, then it checks that state on rerender and sets focus and tabindex

Reply:
that state will be checked within react's lifestyle which is before the nebula calls the component() to run reactROM.render() and the table rerenders. So it is not working as we want.

Reply:
hmm true. What about adding a state on the index component, then passing that into the table? If a set function is passed as well, then it should be possible to set it on confirm, and it should be true when render runs after the data is updated. It would kind of be a placeholder until the useAccessibility hook is done

Reply:
tried but when setState is called, it will cause the index component to run immediately.

Reply:
I think for 1.0, it is what we could do. I will look at the nebula for phrase 2.

Reply:
I'd say it's better to skip it then. Better to have it lose focus consistently than it keeping focus inconsistently